platform/android/java/editor/proguard-rules.pro

```diff
@@ -0,0 +1,33 @@
+-keep class com.godot.** { *; }
```

Comment:
Should be moved to the `lib` module following the instructions in https://github.com/godotengine/godot/pull/31509#issuecomment-1166700483.